google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -378,6 +378,54 @@ def requester_pays= new_requester_pays
           patch_gapi! :billing
         end
 
+        ##
+        # The Cloud KMS encryption key that will be used to protect files.
+        # For example: `projects/a/locations/b/keyRings/c/cryptoKeys/d`
+        #
+        # @return [String, nil] A Cloud KMS encryption key, or `nil` if none
+        #   has been configured.
+        #
+        # @example
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud::Storage.new
+        #
+        #   bucket = storage.bucket "my-bucket"
+        #
+        #   # KMS key ring should use the same location as the bucket.
+        #   kms_key_name = "projects/a/locations/b/keyRings/c/cryptoKeys/d"
+        #   bucket.default_kms_key = kms_key_name
+        #
+        #   bucket.default_kms_key #=> kms_key_name
+        #
+        def default_kms_key
+          @gapi.encryption && @gapi.encryption.default_kms_key_name
+        end
+
+        ##
+        # Set the Cloud KMS encryption key that will be used to protect files.
+        # For example: `projects/a/locations/b/keyRings/c/cryptoKeys/d`
+        #
+        # @param [String] new_default_kms_key New Cloud KMS key name
+        #
+        # @example
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud::Storage.new
+        #
+        #   bucket = storage.bucket "my-bucket"
+        #
+        #   # KMS key ring should use the same location as the bucket.
```

Comment:
Small nit in documentation: @quartzmo, change the word "should" to "must" for key ring location.

Reply:
Updated for all mentions.